.github/workflows/ci_cd.yml

```diff
@@ -293,10 +293,10 @@ jobs:
         with:
           max_attempts: 2
           retry_on: error
-          timeout_minutes: 50
+          timeout_minutes: 60
```

Comment:
@SMoraisAnsys Should not be the opposite? timeout_minutes 180 and pytest timeout 60 ?